include/zephyr/dt-bindings/regulator/rpi_pico.h

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) 2023 TOKITA Hiroshi <tokita.hiroshi@fujitsu.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_REGULATOR_RPI_PICO_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_REGULATOR_RPI_PICO_H_
+
+#define REGULATOR_RPI_PICO_MODE_HI_Z 0x2
```

Comment:
Minor nit- I would like to see a mode defined for "HI Z disabled", something like `REGULATOR_RPI_PICO_MODE_RUN`. Thi way, the `allowed-modes` and `initial-mode` properties don't have to simply be set to 0, and a user looking to change the regulator mode will have an easier time understanding that their options are RUN mode or HI Z mode

Reply:
Added `REGULATOR_RPI_PICO_MODE_NORMAL`.
(I think the terminology of the `RUN` cause a complex situation in case if not enable `regulator-boot-on` property. So I chose `NORMAL` for it.)